src/class04/route.js

```diff
@@ -0,0 +1,11 @@
+var route1 = 5
```

Comment:
Replace this excerpt with the following exercise:

Write a program to read 3 integer values ​​and write the largest one.
Note that the user will not enter equal values.